wiki/Community/Forum/Off-Topic/en.md

```diff
@@ -7,3 +7,74 @@ stub: true
+
+The term was first used by Ephemeral in the [ITT: We're Dirty](https://osu.ppy.sh/community/forums/topics/40298) thread, and subsequently a few time after. Since then others people in the subforum have adopted the term.
+
+#### Old Avatar Week
+
+Old Avatar Week as an annual event in celebration of the upcoming new year. On December 28 of every year people would change their avatar to one of their older one for one week. This would remind OT regulars of an older time when the person had the avatar, giving a sense of reflection between current times and older times like the transition from the old year to the new year.
+
+#### Dancing anime girls
+
+People in the early off-topic days occasionally created threads dedicated to dancing anime girls. If a thread was bad then posting a few dancing anime girls made it better. When people started making threads about quitting OT and actually not quitting, people took that as attention grabbing and posted dancing anime girls to change thread's topic. Eventually posting dancing anime girls on any goodbye or welcome thread was a common thing to do, and thus solidifying as a tradition.
+
+On 9 June 2017 Ephemeral asserted slightly different version of the tradition into the [off-topic rules](https://osu.ppy.sh/community/forums/topics/604424) by requiring people to post no less than 3 dancing anime girls on birthday threads. This made its way to the osu!'s official rules up until 11 November 2019 when it was removed in a cleanup effort. As of writing the rule still stands in the off-topic subforum's rules.
+
+
+#### Exile
+
+Exile is an unofficial moderation concept introduced by Ephemeral in [the grand edict thread](https://osu.ppy.sh/community/forums/topics/604424) in response to people repeatedly spamming the subforum and creating otherwise annoying content. The exile system is enforced by moderation by removing any thread, and sometimes posts, by the person who is exiled. It serves as an indicator that the person is on the verge of being heavily sanctioned by more official means of moderation.
+
+#### Denizen
+
+Since the off-topic subforum is composed of mostly the same regulars chatting among themselves in threads, everyone comes to know one another other. Those who stick around long enough to be considered regulars are called "denizen". While this term was first used by Ephemeral in [this thread](https://osu.ppy.sh/community/forums/posts/1607086), it was not adopted by regulars until its popularization by the [OT!News thread series](https://osu.ppy.sh/community/forums/topics/577518).
+
+#### OT Lore
+
+OT Lore is an effort started by abraker and perpetuated by other denizens to create a story narrative consisting out of various threads people in OT create. The culmination of this effort was compiled into the [OT!Chronicles](https://osu.ppy.sh/community/forums/posts/6230570), and shaped a time period of OT which consisted of many improvised roleplay related threads.
```

Comment:
when you refer to a username, it's way better if you link to the osu! profile, optionally -- but not mandatory, the choice is yours -- with a flag beforehand:

```md
Now ![][flag_IL] [Ezekiel](https://osu.ppy.sh/users/2198472) saw the wheel in a wheel

...

[flag_IL]: /wiki/shared/flag/IL.gif "Israel"
```